modules/nodes-pods-configuring-pod-critical.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-configuring.adoc
+
+[id='nodes-pods-configuring-critical_{context}']
+= Preventing pod removal using critical pods
+
+There are a number of core components, such as Heapster and DNS, that are critical to a fully functional cluster,
+but, run on a regular cluster node rather than the master. A cluster may stop working properly if a critical add-on is evicted.
+
+The descheduler does not remove critical pods.
+
+,Procedure
+
+To make a pod critical: 
+
+. Create a pod specification or edit existing pods to include the `system-cluster-critical` priority class:
++
+----
+spec:
+  template:
+    metadata:
+      name: critical-pod
+    priorityClassName: system-cluster-critical <1>
+----
+<1> Default priority class for pods that should never be evicted from a node. 
```

Comment:
actually that would be the `system-node-critical` priority.

we also need to cover creating non-default priorities since customers with rarely need to define `node` or `cluster` critical pods, but might want to express priority among their own workloads.

https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/#priorityclass